Marlin/Marlin_main.cpp

```diff
@@ -1305,7 +1305,11 @@ static void engage_z_probe() {
 static void retract_z_probe() {
   // Retract Z Servo endstop if enabled
   #ifdef SERVO_ENDSTOPS
-    if (servo_endstops[Z_AXIS] > -1) {
+    if (servo_endstops[Z_AXIS] > -1)
+    {
+        do_blocking_move_to(current_position[X_AXIS], current_position[Y_AXIS], Z_RAISE_AFTER_PROBING);
```

Comment:
This probably should be wrapped in the conditional `#if Z_RAISE_AFTER_PROBING`. Although later this setting could become an EEPROM parameter, if that would be useful to anyone.